### PR TITLE
Performance

### PR DESCRIPTION
**Pull request by maaloum**
### In this PR, We

- Add an email column to your owners' table
- Generated 3.594.280 visits using insert statement.
- Generated 2.500.000 owners using insert statement
Ran the following queries

- [ ] EXPLAIN ANALYZE SELECT COUNT(*) FROM visits where animal_id = 4;
     Before 
     
q1
    After
after

- [ ] EXPLAIN ANALYZE SELECT * FROM visits where vet_id = 2;
    Before
id2before
 After 
id2after

- [ ] EXPLAIN SELECT * FROM owners where email = '[owner_18327@mail.com](mailto:owner_18327@mail.com)';
     Before
emailbefore

    After

emailafter

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -63,3 +63,10 @@ CREATE TABLE visits (
     FOREIGN KEY (animal_id) REFERENCES animals (id),
     FOREIGN KEY (vet_id) REFERENCES vets (id)
 );
+
```
Comment (t-yanick) [inline schema.sql:67]: - [ ] Great job working on this milestone and in having the screenshots in your PR. I think you can also add index on the Foreign Key email. A general way to solve performance problems like the one given in this project is to add indexes for FOREIGN KEYS or on columns frequently used in queries. 

**Below is example snippet** 
```sql
 CREATE INDEX ON visits (animal_id);
 CREATE INDEX ON visits (vet_id);
 CREATE INDEX ON owners (email);
```

File: data.sql [hunk 2]
```diff
@@ -76,3 +76,38 @@ VALUES	(1,1,1,'2020-05-24'),
...
+INSERT INTO visits (animal_id, vets_id, visit_date ) VALUES (9, 2, '2020-08-03');
+INSERT INTO visits (animal_id, vets_id, visit_date ) VALUES (10, 3, '2020-05-24');
+INSERT INTO visits (animal_id, vets_id, visit_date ) VALUES (10, 1, '2021-01-11');
+
+
+
+-- This will add 3.594.280 visits considering you have 10 animals, 4 vets, and it will use around ~87.000 timestamps (~4min approx.)
+INSERT INTO visits (animal_id, vets_id, visit_date) SELECT * FROM (SELECT id FROM animals) animal_ids, (SELECT id FROM vets) vets_ids, generate_series('1980-01-01'::timestamp, '2021-01-01', '4 hours') visit_timestamp;
+-- This will add 2.500.000 owners with full_name = 'Owner <X>' and email = 'owner_<X>@email.com' (~2min approx.)
+insert into owners (full_name, email) select 'Owner ' || generate_series(1,2500000), 'owner_' || generate_series(1,2500000) || '@mail.com';
```
Comment (t-yanick) [inline data.sql:113]: - [ ] Your BEFORE execution times for the first and third queries are significantly lower than 1000ms. I suggest running the populate database query a few times more as described in the [project requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_performance_audit.md#:~:text=4%20Depening%20on,bigger%20than%201000ms.)

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi @maaloum and @Bornittah , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ichala — APPROVED** [review]
Hi Student  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**